crates/handlers/src/compat/login.rs

```diff
@@ -294,6 +472,10 @@ pub(crate) async fn post(
     };
 
     repo.save().await?;
+    info!(
+        "Session and tokens saved successfully for user: {}",
+        user.username
+    );
```

Comment:
When login we pass a device_id unique to each user so we have a unique session per user coming from our web platform (otherwise each login to the platform would stack as a new session in Element and ask to be checked again which would be a mess). Since it's actually in [the Matrix specification](https://spec.matrix.org/v1.12/client-server-api/#relationship-between-access-tokens-and-devices), I felt like it was actually a missing piece from the compat create.